docs/user/reference/git-integration.rst

```diff
@@ -165,6 +176,31 @@ so that you can log in to Read the Docs with your connected account credentials.
           but there isn't a more granular permission
           that only allows setting up SSH keys for read access.
 
+   .. tab:: GitHub App
+
+      Read the Docs requests the following permissions when connecting your Read the Docs account to our :ref:`GitHub App <reference/git-integration:GitHub App>`.
+
+      Account email addresses (read only)
+          We ask for this so we can verify your email address and create a Read the Docs account.
+
+      When installing the Read the Docs GitHub App in a repository, you will be asked to grant the following permissions:
+
+      Repository permissions
+        Commit statuses (read and write)
```

Comment:
Should we add Checks to this doc and update the permissions so we can use them in the future? (https://github.com/readthedocs/readthedocs.org/issues/12022)

Reply:
There is a message in slack about this, but only Anthony commented on it https://readthedocs.slack.com/archives/C02DNG2HSNP/p1755708072781799